src/Network/Network/Cortex/VirtualHub/NewAzureRmVirtualHubCommand.cs

```diff
@@ -85,14 +85,19 @@ public class NewAzureRmVirtualHubCommand : VirtualHubBaseCmdlet
 
         [Parameter(
             Mandatory = false,
-            HelpMessage = "The route table associated with this Virtual Hub.")]
-        public PSVirtualHubRouteTable RouteTable { get; set; }
+            HelpMessage = "The route tables associated with this Virtual Hub.")]
+        public PSVirtualHubRouteTable[] RouteTable { get; set; }
 
         [Parameter(
             Mandatory = false,
             HelpMessage = "A hashtable which represents resource tags.")]
         public Hashtable Tag { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The sku of the Virtual Hub.")]
+        public string Sku { get; set; }
```

Comment:
Can you add `ValidateSet` with possible SKU values?

Reply:
For now only 2 values have been finalized, but more values will be made available soon and we want to avoid having strict checks on the SKU right now.

Reply:
In that case you can you `PSArgumentCompleter`. It will help autocomplete possible values, but will also allow other values

Reply:
done